src/hooks/protections/received_event.cpp

```diff
@@ -376,6 +391,151 @@ namespace big
 		// clang-format on
 	}
 
+	bool scan_play_sound_event(player_ptr plyr, rage::datBitBuffer& buffer)
+	{
+		bool is_entity = buffer.Read<bool>(1);
+		std::int16_t entity_net_id;
+		rage::fvector3 position;
+
+		if (is_entity)
+		{
+			entity_net_id = buffer.Read<uint16_t>(13);
+		}
+		else
+		{
+			position.x = buffer.ReadSignedFloat(19, 1337.0f);
+			position.y = buffer.ReadSignedFloat(19, 1337.0f);
+			position.z = buffer.ReadFloat(19, 1337.0f);

```

Comment:
I’m guessing this is done to put it somewhere the player can’t hear it? 8000/8000/8000 would be a better location.